.github/workflows/test.yml

```diff
@@ -227,9 +227,10 @@ jobs:
           for dir in $(ls -d test-*); do
               name=${dir##test-}
               echo "Running $name..."
-              if [ "$name" = "a-to-f" ]; then
+              if [ "$name" = "diamond" ]; then
                   SERVER=$BOOSTER_DEV ./runDirectoryTest.sh test-$name --time
                   SERVER=$KORE_RPC_DEV ./runDirectoryTest.sh test-$name --time
```

Comment:
We should probably keep running the `a-to-f` test with both `booster-dev` and `kore-rpc-dev` ?

Reply:
Done.